app/scripts/modules/core/src/authentication/userMenu/userMenu.directive.spec.js

```diff
@@ -1,17 +1,18 @@
 'use strict';
 
 import {SETTINGS} from 'core/config/settings';
+import {mock} from 'angular';
```

Comment:
Try to follow https://github.com/spinnaker/deck/wiki/Conventions#imports for imports. It's kind of a pain because there's no good tooling (that any of us have found) to keep the imports clean and sorted, so we just yell at each other in pull requests.